### ISSUE TITLE
ForceScheduler builderNames cannot be renderable

### ISSUE THREAD

**Issue by martinhanzik** [buildbot/buildbot#4252]
http://docs.buildbot.net/latest/manual/cfg-schedulers.html#configuring-schedulers shows that builderNames can be renderable, looks like BaseScheduler handles that, but ForceScheduler does its own check before calling BaseScheduler.__init__

**Comment by tardyp** [comment]
It does not make much sense to me for a Forcescheduler to have its list of builder renderable.
What is your use case?

**Comment by martinhanzik** [comment]
I'm still trying to figure out what exactly I want to do, but I need to build multiple branches on multiple platforms and I wanted to have a builder per branch per platform (eventually triggered automatically) and have checkboxes/dropdowns in the force scheduler. I'm not sure whether ForceScheduler should or should not have the list renderable, or whether it would hurt to have it, but it seemed like it should from the documentation.

**Comment by tardyp** [comment]
>it seemed like it should from the documentation.

yeah, forcescheduler is not really a scheduler. The common doc for basescheduler does not really apply (it does not have changefilter for example)

**Comment by martinhanzik** [comment]
I guess it makes sense. Also since BaseScheduler renders the builderNames in the constructor, it's probably too early to use the selected options from the ForceScheduler dialog. We can probably close this then.

**Comment by lancelodliu** [comment]
You can use only one builder which is bond to multiple worker and pass diferent branch and platform parameters to it.

**Comment by martinhanzik** [comment]
But I need to do different things on different platforms, not just pass them along to a build system.
